pytorch_blade/pytorch_blade/compiler/jit/torch/shape_analysis.cpp

```diff
@@ -2796,6 +2795,12 @@ class ShapePropagator : public PropertyPropBase {
         node->output()->setType(IntType::get());
       }
       return true;
+    } else if (
+        node->matches(
+            "torch_blade::fake_quant(Tensor _0, Tensor _1, Tensor _2, int _3, int _4, int _5, int[] _6, bool _7, bool _8, bool _9, bool _10) -> Tensor")) {
```

Comment:
Nit: move to `first_input_type_formula` at Line 904

Reply:
done